Changes

```diff
@@ -150,6 +150,8 @@ Working version
 
 ### Standard library:
 
+- #13862: Make List.sort_uniq keep the first occurrences of duplicates.
```

Comment:
Please add author and reviewer information. Also, I believe this change should/could be marked as "breaking" (with an asterisk `*` instead of a dash `-`).

Reply:
I think I addressed this and the previous comments in my PR from last week.